mapstruct-kotlin/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.kevcodez</groupId>
+    <artifactId>kotlin-mapstruct</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Sample application demonstration the use of Mapstruct and Kotlin</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <kotlin.version>1.1.3-2</kotlin.version>
+        <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
+        <mapstruct.version>1.2.0.CR1</mapstruct.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mapstruct</groupId>
+            <artifactId>mapstruct-jdk8</artifactId>
+            <version>${mapstruct.version}</version>
+            <scope>provided</scope>
```

Comment:
When using `Mappers.getMapper(..)` the classes are not available during runtime

> Exception in thread "main" java.lang.NoClassDefFoundError: org/mapstruct/factory/Mappers
>	at de.kevcodez.MainKt.main(Main.kt:10)
> Caused by: java.lang.ClassNotFoundException: org.mapstruct.factory.Mappers
>	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
>	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
>	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
>	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
>	... 1 more

Changing scope to compile